apps/web/src/app/[locale]/(dashboard)/[wsId]/(workspace-settings)/migrations/migration-dashboard.tsx

```
@@ -136,325 +177,236 @@
+                throw data;
+              }
 
-            if (!res.ok) {
-              setLoading(module, false);
-              setError(module, data);
-              internalError = data?.error;
-              return;
+              internalData.push(...newInternalData);
+              setData('internal', module, internalData, internalData.length);
+            } catch (error) {
+              throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the catch clause.


(lint/complexity/noUselessCatch)